xmppserver/src/main/java/org/jivesoftware/openfire/group/DefaultGroupPropertyMap.java

```diff
@@ -1,19 +1,27 @@
+/*
+ * Copyright (C) 2013-2022 Ignite Realtime Foundation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package org.jivesoftware.openfire.group;
 
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.SQLException;
-import java.util.AbstractSet;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Map.Entry;
-import java.util.Set;
+import java.util.*;
```

Comment:
Style thing, I know, but I don't like wildcard imports.

Reply:
I am not bothered by them in the slightest. :) I can count the number of times that these actually caused an issue for me on the fingers of one hand that belongs to the worst employee of a saw mill.

Reply:
That's fine. It has caused me problems in the past, which is why I try and avoid them.

(At the time, we had one person using IntelliJ and another with Eclipse. From memory Eclipse imported a wildcard which tripped up IntelliJ because of two different classes with the same name (or vice-versa? I forget)